_posts/2023-02-07-release.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Release 2023.07
+author: Priya
+date: 2023-02-11
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Support for Stored Procedures in Salesforce
+Mammoth now enables users to execute queries using stored procedures in the query box.
```

Comment:
Users can now execute stored procedure queries in the query box

Reply:
fixed